src/lib/controllers/config-store/adapters/kubernetes-config-store.ts

```diff
@@ -59,88 +40,96 @@ interface UpdateSecretArgs {
  */
 export class KubernetesConfigStore implements ConfigStore {
 	public static async create(access: ConfigStoreAccess) {
-		const kubeConfig = config.fromKubeconfig(
-			_.get(access.kubernetes, 'kubeConfigPath', './kube/config'),
-		);
-		const client = new Client({ config: kubeConfig });
-		let tnlConfig: tunnel.Config | null = null;
-		if (_.get(access.kubernetes, ['bastion', 'host'])) {
-			const privateKey = await fs.readFile(
-				_.get(access.kubernetes, ['bastion', 'key']),
-			);
-			tnlConfig = {
-				username: _.get(access.kubernetes, ['bastion', 'username']),
-				host: _.get(access.kubernetes, ['bastion', 'host']),
-				dstHost: _.get(access.kubernetes, 'endpoint', 'kubernetes'),
-				port: parseInt(_.get(access.kubernetes, ['bastion', 'port'], 22), 10),
-				dstPort: 443,
-				privateKey: privateKey.toString(),
-				localHost: '127.0.0.1',
-				localPort: 8443,
-				passphrase: _.get(access.kubernetes, ['bastion', 'keyPassword']),
+		const getKubeConfigFile = async (pathOrContent: string) => {
+			if (await fs.exists(pathOrContent)) {
+				return pathOrContent;
+			}
+
+			const tempKubeConfigPath = temp.path();
+			await fs.writeFile(tempKubeConfigPath, pathOrContent);
```

Comment:
Not sure where this file is deleted again?

Reply:
explicitly it is not... I need to think about this.